proxy/proxy.go

```diff
@@ -120,9 +121,13 @@ const (
 )
 
 type OpenTracingParams struct {
-	// Tracer holds the tracer enabled for this proxy instance
+	// OtTracer holds the open telemetry tracer enabled for this proxy instance
+	// DEPRECATED, use TracerWrapper.
 	Tracer ot.Tracer
 
+	// OtelTracer holds the opentracing tracer enabled for this proxy instance
+	OtelTracer trace.Tracer
```

Comment:
Here and elsewhere we keep old and new tracer fields. 
Why not use only OTEL tracer and provide users an adapter for opentracing tracer?

Reply:
Actually we do this from skipper.Run() point of view @lucastt , so we likely can delete proxy.Tracer

Reply:
Yeah, it is possible, I just considered this as something that people using skipper as a library could be using so I wanted to keep backward compatibility. Under the hood the old tracer is converted into a tracerWrapper instance that convert calls from OTEL to OT. 

So yeah, if this part does not need to be backward compatible it would be much better to have a single tracer instance.